pystac_client/client.py

```diff
@@ -40,6 +40,7 @@ def open(
         Args:
             url : The URL of a STAC Catalog. If not specified, this will use the `STAC_URL` environment variable.
             headers : A dictionary of additional headers to use in all requests made to any part of this Catalog/API.
+            media_type : Select search endpoint based on the specified `media_type`
```

Comment:
The `media_type` argument is added to the docs, but it isn't added to the actual function signature. After chatting w/ @matthewhanson, we think that this argument should _not_ be included in the function signature, but the value just hardcoded and passed down to PySTAC (so, how its working now is correct).

This docs line should be removed.